react/features/invite/components/add-people-dialog/web/utils.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import {
+    _decodeRoomURI,
+    getDialInfoPageURL,
+    shouldDisplayDialIn
+} from '../../../functions';
+
+/**
+ * Tries to copy a given text to the clipboard.
+ *
+ * @param {string} textToCopy - Text to be copied.
+ * @returns {boolean}
+ */
+export function copyText(textToCopy: string) {
+    const fakeTextArea = document.createElement('textarea');
+
+    // $FlowFixMe
```

Comment:
What was the flow error here?

Reply:
IIRC it implied that the `document.body` could be null and that seemed like a weird error to me, hence the `$FlowFixMe`. I could add a null check, but it seems redundant to me..